File: 3005. Count Elements With Maximum Frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int, int> map;

        for(int i=0;i<nums.size();i++){
            map[nums[i]]++;
        }
        int mx = -1;
        
        for(auto it : map){
            if(it.second >= mx){
                mx = it.second;
            }
        }

        int cnt = 0;
        for(auto it: map){
            if(it.second == mx){
                cnt += it.second;
            }
        }


        return cnt;
    }
};

File: 3005. Count Elements With Maximum Frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        mp = {}
        for i in nums:
            if i in mp:
                mp[i] += 1
            
            else :
                mp[i] = 1
        
        mx = 0
        for val in mp.values():
            if val >= mx:
                mx = val
        
        sm = 0
        for val in mp.values():
            if val == mx:
                sm += mx
        
        return sm
        
